sda-orch/files/README.md

```diff
@@ -5,3 +5,5 @@ This folder should contain the TLS certificates for orchestartor service and the
 - ca.crt, root ca certificate.
 - orch.crt, service certificate.
 - orch.key, service key.
+
+If `useCustomConfig` is enabled it expects a file with the name `config.json` so that it can load it as configuration from it.
```

Comment:
An example custom config would be nice to have.

Reply:
base config: https://github.com/neicnordic/sda-orchestration/blob/61f11f20f50ae830f4a9617249d8dd5630be54dd/sda_orchestrator/config/config.json although the REMS form can become pretty complex, and even datacite related things can become pretty complex in configuration. 

I can mention it as an example, but not sure we can have a flexible template out of it, at least not in this form. I think if we can decouple some of the information and get it from metadata service that would be ideal.